Activate parallel builds under Windows.

danielfenner opened the pull request:
With this flag I get a decent number cl.exe's running in parallel.
It seems to be disabled by default since it is incompatible with
some esoteric stuff we don't use.

I triggered an entire rebuild and things work fine.

pierricgimmig reviewed (commented):
Cool, thanks for this @danielfenner ! It seems there's an issue when building Orbit.rc.res, /MP is then an invalid option. I think we could use cmake's `remove_definitions` maybe? https://cmake.org/cmake/help/latest/command/remove_definitions.html

danielfenner commented:
Funny, the rebuild I trigger locally does not seem to trigger that part of the build!? (I deleted the entire build dir...)

Removed the flag from the "Orbit" sub-CMakeList.txt - let's see if this fixes it on the CI.

Maybe Henning has a more elegant idea where to put this. CXX_FLAGS maybe?

pierricgimmig reviewed (approved):
Cool, lgtm! Thanks @danielfenner

beckerhe commented on CMakeLists.txt:
```diff
@@ -125,6 +125,7 @@ add_definitions(-DUNICODE -D_UNICODE)
 
 if(WIN32)
   add_definitions(-DWIN32)
+  add_definitions(/MP)
```
`string(APPEND CMAKE_CXX_FLAGS " /MP")` should work

beckerhe replied:
And Nit: You should check for MSVC, not for WIN32 in that case:

```
if (MSVC)
  string(APPEND...)
endif()
```

beckerhe reviewed (approved):
Great that it helps
